chapters/ch40.md

```diff
@@ -1,104 +1,103 @@
 > # 40. Generalizing from the training set to the dev set
 
---> _replace THIS LINE by your translation for the above line_
+--> # 40. Tổng quát hóa từ tập huấn luyện đến tập phát triển
```

Comment:
Thay "THIS LINE" là thay "CẢ DÒNG", bao gồm cả '-->' nữa.

Reply:
Em đã relsove rồi a nhé.